_episodes/07-kokkos-openmp.md

```diff
@@ -1,13 +1,17 @@
 ---
 title: "Kokkos with OpenMP"
-teaching: 0
-exercises: 0
+teaching: 30
+exercises: 60
```

Comment:
In our last call discussed cutting back on the requirement to create a complete scalability study and just ask them to fill a couple of gaps in the table. For that reason I think 60 minutes is too much time here, 15 is more likely.

Reply:
Agreed. This is based on a first real estimate from Prithwish, I wouldn't consider these times as fixed.